oidc/views.py

```diff
@@ -52,17 +60,18 @@ def dispatch(self, request, helper):
         return helper.next_step()
 
 
-class OIDCConfigureView(ConfigureView):
-    def dispatch(self, request, organization, auth_provider):
-        config = auth_provider.config
-        if config.get("domain"):
-            domains = [config["domain"]]
-        else:
-            domains = config.get("domains")
-        return self.render(
-            "oidc/configure.html",
-            {"provider_name": ISSUER or "", "domains": domains or []},
-        )
+def oidc_configure_view(
+    request: HttpRequest, organization: RpcOrganization, auth_provider: RpcAuthProvider
+) -> DeferredResponse:
+    config = auth_provider.config
+    if config.get("domain"):
+        domains: list[str] | None
+        domains = [config["domain"]]
+    else:
+        domains = config.get("domains")
+    return DeferredResponse(
+        "oidc/configure.html", {"provider_name": ISSUER or "", "domains": domains or []}
```

Comment:
```suggestion
        "oidc/configure.html",
        {"provider_name": ISSUER or "", "domains": domains or []},
```

Just for consistency with the previous code.

Reply:
ok